code/modules/roguetown/roguejobs/blacksmith/anvil_recipes/valuables.dm

```diff
@@ -13,12 +13,6 @@
 	created_item = /obj/item/clothing/ringP
 	additional_items = list(/obj/item/clothing/neck/roguetown/psicross)
 
-/datum/anvil_recipe/valuables/silver/rings
```

Comment:
This is not a removal of the silver rings. This is a duplicate entry that I am removing. There is a second class/obj definition right below.